kickstart/provision_rhel_atomic.erb

```diff
@@ -0,0 +1,27 @@
+<%#
+kind: provision
+name: RHEL Atomic Kickstart
+%>
+lang en_US.UTF-8
+keyboard us
+timezone America/New_York
+
+# Partition table should create /boot and a volume atomicos
+<%= @host.diskLayout %>
```

Comment:
Perhaps support the %pre/scripted stuff that's in the standard provision script too.